### ISSUE TITLE
currency change 

### ISSUE THREAD

**Issue by loanduong02** [horilla-opensource/horilla#339]
Is there any way to change the currency to match my country? 
Have you developed other currencies?

**Comment by horilla-opensource** [comment]
Hi @loanduong02 ,
You can have your country currency in the system. Just go the _Settings_ > _General_ > _General Settings_ > _Currency_
Paste in your countries currency symbol in the field and save the changes.

**Comment by loanduong02** [comment]
> Hi @loanduong02 , You can have your country currency in the system. Just go the _Settings_ > _General_ > _General Settings_ > _Currency_ Paste in your countries currency symbol in the field and save the changes.

i know but it doesn't change the format like the currency in my country for example 6,000,000 VND

**Comment by horilla-opensource** [comment]
Hi @loanduong02 ,
Rather than prefix terms, you wanted to have postfix right?

**Comment by loanduong02** [comment]
> Hi @loanduong02 , Rather than prefix terms, you wanted to have postfix right?

yes and I want to format as standard in my country for example 6,000,000 VND. Because currently your format only has a dot separating the units

**Comment by loanduong02** [comment]
![image](https://github.com/user-attachments/assets/fe7d3f9f-dba2-40fb-a679-6ea8ff486403)
Why is the basic salary 50 but deducted to 24?

**Comment by horilla-opensource** [comment]
Hi @loanduong02 ,
The basic pay was calculated based on the number of paid days, ie for 12 days. So it was deduced to 24 (contract wage type should be monthly for this calculation).

_60 basic pay for 30 days. So per day 2, Paid days = 12, Salary = 12*2=24_

**Comment by loanduong02** [comment]
> Hi @loanduong02 , The basic pay was calculated based on the number of paid days, ie for 12 days. So it was deduced to 24 (contract wage type should be monthly for this calculation).
> 
> _60 basic pay for 30 days. So per day 2, Paid days = 12, Salary = 12*2=24_

Can you explain the salary calculation part as I mentioned above? Thank you very much

**Comment by horilla-opensource** [comment]
Hi @loanduong02 ,

We have added the feature for having the currency symbol in the postfix or prefix as required by the user.
You can find the option in the settings.

With Regards,
**Team Horilla**

**Comment by loanduong02** [comment]
@horilla-opensource 
Currently I want a thousandth separator, do you plan to develop it in the future? Present, I found a solution as shown in the picture. Is there any way to supplement it effectively?
![image](https://github.com/user-attachments/assets/4bd898bb-15e1-4044-946f-5ff1d54d0707)

**Comment by horilla-opensource** [comment]
Hi @loanduong02,

Thanks for your feedback!

Yes, we are planning to add better support for thousand separators and currency formatting in future updates. We'll keep you posted once it's available.

Appreciate your patience and great suggestions!

With Regards,
**Team Horilla**
